docs/book/src/developer/providers/implementers-guide/building_running_and_testing.md

```diff
@@ -32,15 +32,15 @@ Before you can deploy the infrastructure controller, you'll need to deploy Clust
 
 You can use a precompiled manifest from the release page, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`.
 
-``` shell
+```bash
 cd cluster-api
 kustomize build config/ | kubectl apply -f-
 ```
 
 Check the status of the manager to make sure it's running properly
 
-```shell
-$ kubectl describe -n capi-system pod | grep -A 5 Conditions
+```bash
+kubectl describe -n capi-system pod | grep -A 5 Conditions
```

Comment:
could yout split `kubectl describe -n capi-system pod | grep -A 5 Conditions` from the output

Reply:
Done